src/neuroconv/tools/roiextractors/roiextractors.py

```diff
@@ -371,6 +374,96 @@ def image_mask_iterator():
     return nwbfile
 
 
+def add_fluorescence(segmentation_extractor: SegmentationExtractor, nwbfile: NWBFile, metadata: dict) -> NWBFile:
```

Comment:
I think this should be called `add_fluoresence_traces`. Adding the Fluoresence MultiContinaer is a only a small part of what this function does. What do you think?